Tests/test_image_access.py

```diff
@@ -407,11 +407,6 @@ def test_putpixel_overflow_error(self, mode):
         with pytest.raises(OverflowError):
             im.putpixel((0, 0), 2**80)
 
-    def test_putpixel_unrecognized_mode(self):
-        im = hopper("BGR;15")
-        with pytest.raises(ValueError, match="unrecognized image mode"):
-            im.putpixel((0, 0), 0)
```

Comment:
With the changes in this PR, you can no longer convert to an image mode that you can't create directly.